src/main/kotlin/org/rescado/server/config/SecurityConfig.kt

```diff
@@ -28,7 +28,7 @@ class SecurityConfig(
             /**/.sessionCreationPolicy(SessionCreationPolicy.STATELESS).and()
             .authorizeRequests()
             /**/.anyRequest().permitAll().and()
-            .regexMatcher("^(?!\\$AUTH_ROUTE|\\$INFO_ROUTE).*")
+            .regexMatcher("^(?!\\$AUTH_ROUTE|\\$PUBLIC_ROUTE).*")
```

Comment:
The idea here is that in the future we might want more endpoints publicly available. With this change the only requirement is to put them under `/api/public/**`.